src/mcts/search.cc

```diff
@@ -218,8 +234,10 @@ inline float ComputeCpuct(const SearchParams& params, uint32_t N,
 }  // namespace
 
 std::vector<std::string> Search::GetVerboseStats(Node* node,
-                                                 bool is_black_to_move) const {
-  const float fpu = GetFpu(params_, node, node == root_node_);
+                                                 bool is_black_to_move,
+                                                 bool is_odd_depth) const {
```

Comment:
nit: if `GetVerboseStats` is only used at root or ponder, seems like the existing and added `bool`s can be derived with some checks that include `node == root_node_`. Otherwise calls with bool arguments can get confusing, e.g., `GetVerboseStats(node, false, true);` does what again (pretend you weren't looking at the parameters here 😛 )

Reply:
Done.